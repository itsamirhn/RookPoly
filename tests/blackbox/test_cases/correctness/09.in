5
01110
10101
11011
10101
01110
