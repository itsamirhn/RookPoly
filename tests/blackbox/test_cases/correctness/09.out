4X^5 + 20X^4 + 36X^3 + 28X^2 + 9X + 1
